lightning/src/events/bump_transaction/mod.rs

```diff
@@ -528,6 +528,15 @@ where
 				);
 				return Err(());
 			}
+			while !selected_utxos.is_empty()
+				&& selected_amount - selected_utxos.front().unwrap().0.output.value
+					>= target_amount_sat + total_fees
```

Comment:
Should this be `total_fees - selected_utxos.front().unwrap().1` to also remove the fees for this utxo if we're going to drop it?